rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java

```diff
@@ -193,45 +198,58 @@ Canvas build() {
 
         this.nodeLineSeriesForm = new DynamicForm();
         nodeLineSeriesForm.setHeight(24);
-        nodeLineSeriesForm.setNumCols(12);
+        nodeLineSeriesForm.setNumCols(16);
         nodeLineSeriesForm.setWidth(300);
         CheckboxItem freeIt = new CheckboxItem("free",
                                                "<span style='background:#35a849;'>&nbsp;&nbsp;&nbsp;</span> Free");
         freeIt.setValue(true);
-        freeIt.setWidth(120);
+        freeIt.setWidth(90);
```

Comment:
Set final variable for width instead of modifying each width one by one.

Reply:
true